File: app/src/main/java/com/neome/java/api/meta/base/dto/StudioEntRoleMap.java
// neome.ai API - do not change
//

package com.neome.java.api.meta.base.dto;

import com.neome.java.api.meta.base.Types.MetaIdRole;

import java.util.Map;

@SuppressWarnings("unused")
public class StudioEntRoleMap extends StudioBase
{
  public MetaIdRole[] keys;

  public Map<MetaIdRole, StudioEntRole> map;
}

File: app/src/main/java/com/neome/api/meta/base/dto/StudioEntRoleMap.kt
// neome.ai API - do not change
//
@file:Suppress("unused", "MemberVisibilityCanBePrivate", "CanBeParameter", "PropertyName")


package com.neome.api.meta.base.dto

import com.neome.api.meta.base.Types.MetaIdRole
import java.util.Map

open class StudioEntRoleMap : StudioBase() {
    lateinit var keys: Array<MetaIdRole>
    lateinit var map: Map<MetaIdRole, StudioEntRole>
}
